repository/kafka/docs/latest/versions.md

```diff
@@ -3,3 +3,8 @@
 - Apache Kafka 2.12-2.3.0
 - OpenJDK 8
 - JMX Exporter 0.1.0
+
+
+### KUDO Version Requirement
+
+KUDO version 0.4.0 or later
```

Comment:
is there something in kafka that will not work with 0.3.3? I think it does not matter at this point so leave it as is, just curious...

Reply:
we just want users to not to lock in `0.3.3` for example. So adding as a requirement here.